addons/mrp/models/stock_orderpoint.py

```diff
@@ -42,6 +42,19 @@ def _compute_show_bom(self):
         for orderpoint in self:
             orderpoint.show_bom = orderpoint.route_id.id in manufacture_route
 
+    def _get_min_qty_order_possible(self):
+        """
+        Calculates the minimum quantity that can be ordered according to the qty and UoM of the BoM
+        """
+        self.ensure_one()
+        min_qty_order_possible = super()._get_min_qty_order_possible()
+        if self.route_id:
+            rule_action = self.route_id.rule_ids[0].action
+            if rule_action == "manufacture" and self.product_id.bom_ids:
+                bom = self.product_id.bom_ids[0]
```

Comment:
It will traceback if the user checked manufacture on its product but doesn't create a BoM for it